pkg/controller/studyjob/katib_api_util.go

```diff
@@ -328,3 +355,38 @@ func getSuggestion(c katibapi.ManagerClient, studyID string, suggestionSpec *kat
 	}
 	return getSuggestReply, nil
 }
+
+func validateSuggestionParameters(c katibapi.ManagerClient, studyConfig *katibapi.StudyConfig, suggestionParameters []katibapi.SuggestionParameter, suggestionAlgorithm string) bool {
+	ctx := context.Background()
+
+	validateSuggestionParametersReq := &katibapi.ValidateSuggestionParametersRequest{
+		StudyConfig:         studyConfig,
+		SuggestionAlgorithm: suggestionAlgorithm,
+	}
+
+	for _, p := range suggestionParameters {
+		validateSuggestionParametersReq.SuggestionParameters = append(
+			validateSuggestionParametersReq.SuggestionParameters,
+			&katibapi.SuggestionParameter{
+				Name:  p.Name,
+				Value: p.Value,
+			},
+		)
+	}
+
+	_, err := c.ValidateSuggestionParameters(ctx, validateSuggestionParametersReq)
+
+	statusCode, _ := status.FromError(err)
+
+	if statusCode.Code() == codes.Unknown {
+		log.Printf("Method ValidateSuggestionParameters not found inside Suggestion service: %s", suggestionAlgorithm)
+		return true
+	}
```

Comment:
Please handle `InvalidArgument` and `Unavailable` error and print the reason for the error (the parameter is invalid or service is unavailable).